Beds 1494/split autowithdrawals frontend

enzo-bitfly opened the pull request.

marcel-bitfly commented on frontend/components/dashboard/table/DashboardTableClWithdrawals.vue:
```diff
@@ -174,7 +174,9 @@ const getGroupName = (groupId: number) => {
                 <BcTooltip
-                  v-if="slotProps.data.type === 'auto' && slotProps.data.status === 'queued'"
+                  v-if="
+                    (slotProps.data.type === 'skimming' || slotProps.data.type === 'system')
+                      && slotProps.data.status === 'queued'"
```
**question:** @remoterami can you give us a source where we can read about that, to understand what is happening here, it gets confusing a bit

marcel-bitfly commented on frontend/i18n/locales/en.json:
```diff
@@ -266,6 +266,8 @@
                 "info_auto_queued": "Only shows the next automatic withdrawal estimate. Be aware that manual withdrawals use a different queue.",
                 "info_slot_processed_estimated": "This is an estimation since this withdrawal is still queued.",
+                "info_type_skimming": "This was an automatic skimming withdrawal because your validators balance exceeded its maximum effective balance.",
+                "info_type_system": "This special withdrawal was triggered by switching to a compounding validator. The amount was not withdrawn but automatically re-deposited, viewable in the Deposits tab.",
```
**typo:** 
```suggestion
                "info_type_system": "This special withdrawal was triggered by switching to a compounding validator. The amount was not withdrawn but automatically re-deposited, viewable in the deposits tab.",
```

enzo-bitfly replied:
done

marcel-bitfly commented:
small note:
```feat: split automatic cl withdrawal types```
Without more context this`commit message` is hard to understand (this would be a case where it would be great to explain what this is about in the body, if you stick with this `commit message title`)

enzo-bitfly committed:
feat: split automatic `cl withdrawal types`

- `auto cl withdrawals` can be caused either by `skimming` rewards or by `re-staking` them in case of a compounding validator

See: BEDS-1494

enzo-bitfly committed:
feat(DashboardTableClDeposits): increase detail footer's `line-height`

See: BEDS-1350
